libcaf_core/test/flow/mixed.cpp

```diff
@@ -69,6 +69,31 @@ SCENARIO("sum up all the multiples of 3 or 5 below 1000") {
   }
 }
 
+SCENARIO("take the last 5 digits in a series of 100 digits") {
```

Comment:
Please move these tests to a new `caf/flow/observable.test.cpp` that is using the new testing framework. The tests are ok, but they operate on the blueprints. There should be at least one additional test that calls `take_last` on an actual `observable`.